runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -206,46 +205,52 @@ public NettyClientRouter(String endpoint) {
      * @param host Host to connect to.
      * @param port Port to connect to.
      */
+    @Deprecated
     public NettyClientRouter(String host, Integer port) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)